applications/nrf_desktop/sysbuild/CMakeLists.txt

```diff
@@ -8,10 +8,6 @@
 set(SB_APPLICATION_CONFIG_DIR "${CMAKE_CURRENT_LIST_DIR}/../configuration/\${NORMALIZED_BOARD_TARGET}")
 
 # Redirect images' configuration directories to use those in the board configuration directory
```

Comment:
Consider expanding comment why the default image is not handled in here like remaining images

Reply:
Actually that's how most of the apps handle `APPLICATION_CONFIG_DIR`. I think there is no need to provide further clarification here (approach with `nrf_desktop_APPLICATION_CONFIG_DIR` defined through sysbuild was specific to our apps).

Reply:
Ok, I will add explanation there.